contracts/credit/CreditConfiguratorV3.sol

```diff
@@ -450,9 +451,12 @@ contract CreditConfiguratorV3 is ICreditConfiguratorV3, ACLNonReentrantTrait {
     {
         if (newPriceOracle == CreditManagerV3(creditManager).priceOracle()) return;
 
+        (uint128 minDebt,) = CreditFacadeV3(creditFacade()).debtLimits();
+        if (IPriceOracleV3(newPriceOracle).convertToUSD(minDebt, underlying) == 0) revert IncorrectPriceException(); // I:[CC-21]
+
         uint256 num = CreditManagerV3(creditManager).collateralTokensCount();
         unchecked {
-            for (uint256 i; i < num; ++i) {
+            for (uint256 i = 1; i < num; ++i) {
```

Comment:
Skipping the underlaying `getPrice` check. The underlaying oracle is separetly checked via `convertToUSD` and verified to not be zero to prevent dust on the account that would have not value.